apps/explorer/src/components/transactions/TransactionsForAddress.tsx

```diff
@@ -64,46 +63,55 @@ export function TransactionsForAddressTable({
     return <TableCard data={data} columns={tableColumns} />;
 }
 
-export function TransactionsForAddress({
-    address,
-    type,
-}: TransactionsForAddressProps): JSX.Element {
+export function TransactionsForAddress({ address }: TransactionsForAddressProps): JSX.Element {
     const client = useIotaClient();
+    const indexerClient = useIotaIndexerClient();
 
     const { data, isPending, isError } = useQuery({
-        queryKey: ['transactions-for-address', address, type],
+        queryKey: ['transactions-for-address', address, !!indexerClient],
```

Comment:
having the indexer client as a query is probably not a good idea performance-wise. All queries are invalidated when the network is changed, this is why we don't put the normal iota client as query key in other queries
```suggestion
        queryKey: ['transactions-for-address', address],
```